src/common/paddle/organization/eventHandler.ts

```diff
@@ -0,0 +1,39 @@
+import { EventName, type EventEntity } from '@paddle/paddle-node-sdk';
+
+import { SubscriptionProcessor, SubscriptionProvider } from '../../plus';
+import { logger } from '../../../logger';
+import { createOrganizationSubscription } from './processing';
+
+export const processOrganizationPaddleEvent = async (event: EventEntity) => {
+  switch (event?.eventType) {
+    case EventName.SubscriptionCreated:
+      await createOrganizationSubscription({ event });
+      break;
```

Comment:
Do you want to wait for transaction completed to make sure payment went through.

Reply:
Will check again what we do for plus